fflas-ffpack/paladin/parallel.h

```diff
@@ -27,7 +27,7 @@
 #ifndef __FFLASFFPACK_fflas_parallel_H
 #define __FFLASFFPACK_fflas_parallel_H
 
-
+#include <stdarg.h>
```

Comment:
Why do you need that?

Reply:
This seems to be necessary for defining variadic functions which means the three dots for the args.

Reply:
Once taken off, I got build failure on Travis so I think I will keep this.

Reply:
Still does not make sense:
- looking at Travis, it does not look like the build failed because of that, but just stalled ;
- macros have nothing to do with included headers ;
- you added nothing different to what there was before in this file, that include could be removed.

Reply:
OK, I will take off this line and try to push again.

Reply:
@Breush  Negative, the build with clang failed again, as this happened so often I don't think the failure caused by some stalled build is completely random.

Reply:
This is just random, because if you check the old travis build https://travis-ci.org/linbox-team/fflas-ffpack/builds/563913587?utm_source=github_status&utm_medium=notification, it worked fine for the PR run.

Reply:
Yes, but this build I am afraid is with the include line

Reply:
UP Still not useful

Reply:
@Breush OK, I will take off the seemingly unnecessary header file again to check if this time it will be built correctly on the travis